optimize/components/userguide/process-KPIs.md

```diff
@@ -0,0 +1,32 @@
+---
+id: process-KPIs
+title: Process KPIs
+description: "Learn how key performance indicators (KPIs) can be created and used in Optimize."
+---
+
+Key performance indicators (KPIs) are reports represented by single numerical values with predefined targets.
+To create a custom KPI, follow the steps below:
+
+1. On the right side of the **Home** page, select **Create New > New Report > Process KPI**.
```

Comment:
Do we mean of the **Collections** page?

Reply:
And **Create new > Report**? Maybe you can just direct me to the demo env 😅

Reply:
@christinaausley you are right, this is a **Collections** page right now. It used to be named **Home** some longer time ago but neither it's called **Home**, nor it is a default page of optimize. Good catch!